CONTRIBUTING.md

```diff
@@ -17,6 +17,107 @@ They prefer those direct inquiries go through direct messages on gitter or by me
 You can see the list of [maintainers](#maintainers) below.
 
 
+## What to Work On
+
+There is *a lot* of work to do on this project, no matter your area of expertise or skill level.
```

Comment:
Maybe though we should also say that we try to keep the core small, and that some/most new features belong in extensions.

Reply:
I am trying to be as motivational as possible in this section. We will dash their hope in the "scope" section, which should probably be expanded as well.